ietf/settings/base.py

```diff
@@ -244,4 +244,4 @@
     "home.IABHomePage",
 ]
 
-NOTE_WELL_REPO = "https://github.com/ietf/note-well/raw/main/note-well.md"
+NOTE_WELL_REPO = "https://raw.githubusercontent.com/ietf/note-well/65ea9057c9e93c027bc4b96183c47b0a8c1c17ea/note-well.md"
```

Comment:
Can you use either https://raw.githubusercontent.com/ietf/note-well/main/note-well.md or https://github.com/ietf/note-well/raw/main/note-well.md ?
We need a URL without a hash so it gets the latest markdown file.